Soccar [Game-Server]/src/nl/soccar/gameserver/model/PlayerWrapper.java

```diff
@@ -0,0 +1,56 @@
+package nl.soccar.gameserver.model;
+
+import nl.soccar.gameserver.model.session.SessionWrapper;
+import nl.soccar.library.Player;
+import nl.soccar.library.Session;
+import nl.soccar.library.enumeration.CarType;
+import nl.soccar.library.enumeration.Privilege;
+import nl.soccar.socnet.connection.Connection;
+
+/**
+ * @author Lesley
+ */
+public final class PlayerWrapper {
+
+    private final Connection connection;
+    private final Player player;
+
+    public PlayerWrapper(Connection connection, Player player) {
+        this.connection = connection;
+        this.player = player;
+    }
+
+    public SessionWrapper getCurrentSession() {
+        Session session = player.getCurrentSession();
+        if (session == null) {
+            return null;
+        }
+
+        return GameServer.getInstance().getSessionController().getSession(session.getRoom().getName());
+    }
+
+    public int getPlayerId() {
+        return player.getPlayerId();
+    }
+
+    public String getUsername() {
+        return player.getUsername();
+    }
+
+    public CarType getCarType() {
+        return player.getCarType();
+    }
+
+    public Privilege getPrivilege() {
+        return player.getPrivilege();
+    }
+
+    public Connection getConnection() {
+        return connection;
+    }
+
+    public Player unwrap() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)